duel/expr.py

```diff
@@ -172,7 +172,7 @@ def __init__(self, n, a1, to):
         self.name_, self.to = n, to
     def eval(self):
         for n1,v1 in self.arg1_.eval():
-            for i in xrange(0 if self.to else v1, v1 if self.to else sys.maxint):
+            for i in range(0 if self.to else v1, v1 if self.to else sys.maxint):
                 v = gdb.Value(i).cast(v1.type)
```

Comment:
1. I believe that python3 doesn't have `sys.maxint`, I'll replace it with an explicit constant 
2. I cannot use `range` in python2, `range(0, sys.maxint)` will try to create a list that long and will run out of memory. I'll use the same workaround I have in `pretty_printer.py`:
```python
try: a=xrange(0,10)
except: xrange=range
```